draft-ietf-jsonpath-base.md

```diff
@@ -911,8 +943,129 @@ size doesn't
 fit in the implementation's representation of an integer, the implementation
 MUST raise an error.
 
+### Descendant Selector
+
+#### Syntax
+{: numbered="false" toc="exclude"}
+
+The descendant selector starts with a double dot `'..'`. It can be used in place of the normal `dot-selector`. So it will usually be followed by an object member name. As an extension to that natural rule it might also be followed by an `index-selector` acting on objects or arrays.
```

Comment:
... this is nearly the wording in E4X - where I stole that operator from. This is quite important:

* Combination of descendant-operator with index operator is valid: `..['key']` as is `..[*]`
* But combination of descendant-operator with dot-operator resulting in `...key` or `...*` is not.

So formulation "can be used in place of the normal `dot-selector`" makes this clear, in order to avoid the latter.

Reply:
The `descendant-selector` and `dot-selector` are completely different selectors that happen to have an overlapping syntax.  After all, `$..['foo']` is valid, but `$.['foo']` is not.

To say that `descendant-selector` can be used "in place of" `dot-selector` conveys replacement, which is incorrect because they do different things.  It's more correct to say that they share a similar syntax in that they both support `dot-member-name`.

To be honest, I'm not sure we need the sentence at all.

```suggestion
The descendant selector starts with a double dot `'..'` and can be followed by an object member name (similar to the `dot-selector`) or by an `index-selector` acting on objects or arrays.
```

Reply:
Resolved in #102

(Of course, we can, and probably will, tweak further.)